### DISCUSSION TITLE
Share connection pool between seaORM and sqlx

### DISCUSSION THREAD

**Discussion by v-morlock** [SeaQL/sea-orm#2260] [Q&A]
Hello, 

is it possible to feed a seaorm DatabaseConnection and DatabaseTransaction into the sqlx::query!() macro? It would be nice to be able to use both seaorm and sqlx::query! in one project without opening two connection pools and without passing two pools around

Thanks!

**Comment by tyt2y3** [comment]
No, it's not possible. but you can borrow the pool with https://docs.rs/sea-orm/1.0.0-rc.4/sea_orm/enum.DatabaseConnection.html#method.get_mysql_connection_pool
